src/main/java/bdv/fx/viewer/render/TransformAwareBufferedImageOverlayRendererFX.java

```diff
@@ -99,6 +100,23 @@ public void drawOverlays(final Consumer<Image> g)
 
 				LOG.trace("Setting image to {}", sourceImage);
 				g.accept(null);
+
+				/*
+				 * NOTE: need to make the final image fully opaque to ensure that it is properly drawn on the screen later on.
+				 * It is only necessary because existing BlendMode options in JavaFX always perform some kind of blending with old contents of the displayed component.
+				 *
+				 * Ideally, we would use BlendMode.SRC if it was available (that would overwrite the destination with the source regardless of alpha).
+				 * See discussion about possibility of adding this mode in the future versions and more information here:
+				 *
+				 * https://bugs.openjdk.java.net/browse/JDK-8092156
+				 *
+				 * https://books.google.com/books?id=UxM2iXiFqbMC&pg=PA97&lpg=PA97&dq=Porter-Duff+%22source+over+destination%22&source=bl&ots=hYj5U2X5Lk&sig=ACfU3U095X67T4FghqCpgYrAhEWbNtd_xQ&hl=en&sa=X&ved=2ahUKEwjyg4rRufXfAhXvuFkKHfK8BysQ6AEwAnoECAcQAQ#v=snippet&q=%22overwrites%20the%20destination%20with%20the%20source%2C%20regardless%20of%20alpha%22&f=false
+				 *
+				 * https://docs.oracle.com/javase/8/javafx/api/javafx/scene/effect/BlendMode.html
+				 */
+				for (final ARGBType px : sourceImage.asArrayImg())
+					px.set(px.get() | (0xff << 24));
```

Comment:
Does the compiler optimize this? I would prefer to have a static final integer for the alpha value, e.g.
```java
private static final int FULL_OPACITY = 0xff << 24;
```

Reply:
Pretty sure constant expressions are optimized, `ARGBType` internals also look the same.
I can add it to better conform to the project's coding style.